draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx

```diff
@@ -21,55 +21,67 @@ export type CollapsibleProps = {
   separated?: boolean
   sticky?: Sticky
   noSectionPadding?: boolean
+  /**
+   * **Deprecated:** Use test id compatible with your testing library (eg. `data-testid`).
+   * @deprecated
```

Comment:
I know this looks strange, but if you start with `@deprecated` (which assists the IDE), Storybook will ignore the whole comment and not show it in the args table, thus it is flipped.

There's an issue open for it https://github.com/storybookjs/storybook/issues/9721